docs/ENV.md

```diff
@@ -13,9 +13,19 @@ You can create a 32-digit secret key from this website: https://generate.plus/en
 #### `CUBEJS_SCHEDULED_REFRESH_TIMER`
 If true, it enables scheduled refreshes. Can also be set to a number representing how many seconds to wait before triggering another refresh. Recommended value is true for production deployments. Defaults to false.
 
-### Database settings
+### `PLIO_ORGANIZATIONS_SCHEMAS`
+##### Postgres
+If you're using Postgres as your Cube.js database, these are the schema names within the database. Run the following query in your postgresql to retrive the list of organization schemas:
+```sql
+SELECT string_agg(schema_name, ',') FROM public.organization;
+```
+
+##### BigQuery
+If you're using BigQuery as your Cube.js database, these are the datasets within your BigQuery project. Use a comma-separated list of all the datasets.
+
+### Database settings (for Postgres)
```

Comment:
from what I understand, we are not using this right? Would be good to say that clearly here.

Reply:
I have updated it. Let me know if it makes sense.